app/api/dao/mentorship_relation.py

```diff
@@ -275,29 +275,29 @@ def cancel_relation(user_id: int, relation_id: int):
         """
 
         user = UserModel.find_by_id(user_id)
-        request = MentorshipRelationModel.find_by_id(relation_id)
+        relation = MentorshipRelationModel.find_by_id(relation_id)
 
         # verify if request exists
-        if request is None:
+        if relation is None:
             return messages.MENTORSHIP_RELATION_REQUEST_DOES_NOT_EXIST, HTTPStatus.NOT_FOUND
 
         # verify if request is in pending state
-        if request.state != MentorshipRelationState.ACCEPTED:
+        if relation.state != MentorshipRelationState.ACCEPTED:
             return messages.UNACCEPTED_STATE_RELATION, HTTPStatus.BAD_REQUEST
 
         # verify if I'm involved in this relation
-        if not (request.mentee_id == user_id or request.mentor_id == user_id):
+        if not (relation.mentee_id == user_id or relation.mentor_id == user_id):
             return messages.CANT_CANCEL_UNINVOLVED_REQUEST, HTTPStatus.BAD_REQUEST
 
         # All was checked
-        request.state = MentorshipRelationState.CANCELLED
-        request.save_to_db()
+        relation.state = MentorshipRelationState.CANCELLED
+        relation.save_to_db()
 
         return messages.MENTORSHIP_RELATION_WAS_CANCELLED_SUCCESSFULLY, HTTPStatus.OK
 
     @staticmethod
     @email_verification_required
-    def delete_request(user_id: int, request_id: int):
+    def delete_request(user_id: int, relation_id: int):
```

Comment:
Same here i.e. no need to do any of the changes(i.e from request_id to relation_id) in this function. Since, we still haven't accepted the relation request.